docs/readthedocs/index.rst

```diff
@@ -10,13 +10,11 @@ Welcome to the documentation of simona - an agent-based discrete-event power sys
 Institute of Energy Systems, Energy Efficiency and Energy Economics at TU Dortmund University, Germany.
 
 .. toctree::
-   :maxdepth: 3
+   :maxdepth: 2
 
-   about
    usersguide
    config/config
-   models/models
-   config/runtime/participant/load
+   models/models.md
    protocols
```

Comment:
What do you think about having models and protocols as sub points of a "Developer's guide"?

Reply:
I like this idea